tgui/packages/tgui-panel/chat/renderer.js

```diff
@@ -242,29 +242,29 @@ class ChatRenderer {
     }
   }
 
-  setHighlight(text, color) {
+  setHighlight(text, color, matchWord, matchCase) {
     if (!text || !color) {
       this.highlightRegex = null;
       this.highlightColor = null;
       return;
     }
-    const allowedRegex = /^[a-z0-9_\-\s]+$/ig;
     const lines = String(text)
       .split(',')
-      .map(str => str.trim())
+      // eslint-disable-next-line no-useless-escape
+      .map(str => str.trim().replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&'))
```

Comment:
this is removing punctuation? escaping it?
Add a comment here explaining the regex since most people can't read regex easily.

Reply:
don't know if you mean github or code comment

but yes, it escapes every character found in the regex by placing a `\` infront every one of them 
list is: `- / \ ^ $ * + ? . ( ) | [ ] { }`

Reply:
code comment